src/Form/Extension/ProductTypeExtension.php

```diff
@@ -4,25 +4,32 @@
 
 namespace Loevgaard\SyliusBrandPlugin\Form\Extension;
 
-use Loevgaard\SyliusBrandPlugin\Form\Type\BrandChoiceType;
+use Loevgaard\SyliusBrandPlugin\Form\Type\BrandAutocompleteChoiceType;
 use Sylius\Bundle\ProductBundle\Form\Type\ProductType;
 use Symfony\Component\Form\AbstractTypeExtension;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\Validator\Constraints\NotNull;
 
 class ProductTypeExtension extends AbstractTypeExtension
 {
+    /**
+     * {@inheritdoc}
+     */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('brand', BrandChoiceType::class, [
+        $builder->add('brand', BrandAutocompleteChoiceType::class, [
             'placeholder' => 'loevgaard_sylius_brand.form.product.select_brand',
             'label' => 'loevgaard_sylius_brand.form.product.brand',
             'required' => false,
         ]);
     }
 
-    public function getExtendedType(): string
+    /**
+     * {@inheritdoc}
+     */
+    public static function getExtendedTypes()
```

Comment:
A few of your methods are missing return types. Try to run a code inspection with this plugin installed: https://github.com/Setono/Coding-Guidelines#6-install-php-inspections-ea-extended

Reply:
Did you see this?